modules/ROOT/pages/configure-acm.adoc

```diff
@@ -172,7 +172,7 @@ NOTE: Remember to associate the external data source with this authentication pr
 . If you don't have an active session in your Anypoint Platform organization, the *Anypoint Login* page is shown. Log in to the Anypoint Platform to complete the authentication. If you already have an active session, authentication is performed without requiring a login.
 . Verify the Exchange *External Data Sources* *Authentication Status* is *Authenticated*.
 
-After authenticating the External Data Source, provide the credentials of an Anypoint Platform system user with read-only access, for components like the API Console which retrieve data directly from the Anypoint Platform APIs.
+After authenticating the External Data Source, provide the credentials of an Anypoint Platform system user with read-only access, for Lightning components like the API Console which retrieve data directly from the Anypoint Platform APIs.
```

Comment:
This is a complex sentence. Can you reword to simplify or maybe break into two sentences?

Reply:
If you use "which," you need to precede it with a comma; if not, use "that".